hhvm/content.md

```diff
@@ -0,0 +1,84 @@
+# What is HHVM?
+
+HHVM is an open-source virtual machine designed for executing programs written in Hack and PHP. HHVM uses a just-in-time (JIT) compilation approach to achieve superior performance while maintaining the development flexibility that PHP provides.
+
+> [wikipedia.org/wiki/HipHop_Virtual_Machine](https://en.wikipedia.org/wiki/HipHop_Virtual_Machine)
+
+![logo](https://raw.githubusercontent.com/baptistedonaux/docker-hhvm/master/logo.png)
+
+# How to use this image
+
+Two methods (CLI vs FastCGI):
+
+## CLI mode
+
+```console
+$ docker run -v /foo.php:/foo.php foo.php
```

Comment:
This line is missing the `hhvm` image to run :wink: `docker run -v /foo.php:/foo.php hhvm foo.php`